<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XResDial</class>
 <widget class="QWidget" name="XResDial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>140</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>140</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="res7Button">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>31</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res2Button">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>60</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res6Button">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>8</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res3Button">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>29</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res8Button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res5Button">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>0</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MYDial" name="dial">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>21</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <number>8</number>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res4Button">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>7</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res1Button">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>83</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res9Button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>83</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Units:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="unitsComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>Âµs</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ms</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MYDial</class>
   <extends>QDial</extends>
   <header>mydial.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
